node-backend/app/routes/badgesRoutes_v1.js

```diff
@@ -1,26 +1,47 @@
 const express = require("express");
 const bodyParser = require("body-parser");
 const BadgesController = require("../controllers/v1/badgesController");
+const jwtValidate = require("../jwtTokenValidation/jwtValidation");
 const badgesRouter = express.Router();
 badgesRouter.use(bodyParser.urlencoded({ extended: true }));
-//for create core value
+
+async function authorizedRole(req, res, next) {
```

Comment:
@jitendrasbunde already written function to decode token and access role from it. Why you written new function?